src/zenml/cli/__init__.py

```diff
@@ -1253,6 +1262,13 @@ def my_pipeline(...):
 zenml connect --username zenml --password=Pa@#$#word --config=/path/to/zenml_server_config.yaml
 ```
 
+You can open the ZenML dashboard of your currently connected ZenML server using
+the following command:
+
+```bash
+zenml show
+```
+
```

Comment:
```suggestion
```

Note that if you have set your `AUTO_OPEN_DASHBOARD` environment variable to `false` then this will not open the dashboard until you set it back to `true`.
```